duckduckgo/iOS
# wire up sync account creation and recovery to backend

Task/Issue URL: https://app.asana.com/0/414235014887631/1204209043038910/f
Tech Design URL:
CC:

**Description**:

Connects the UI to the backend via BSK.



**Steps to test this PR**:

1. See steps in https://github.com/duckduckgo/BrowserServicesKit/pull/271
2. General smoke testing of Sync UI




**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
